wiki/Beatmapping/Approach_Rate/en.md

```diff
@@ -0,0 +1,49 @@
+---
+tags:
+  - AR
+  - reading
+---
+
+# Approach rate
+
+*For suggested AR values, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+**Approach rate** (***AR***) exists only in [osu!standard](/wiki/Game_Modes/osu!) and [osu!catch](/wiki/Game_Modes/osu!catch). AR defines when [hit objects](/wiki/Hit_Objects) start to fade in relatively to when they should be hit or collected. AR values range from 0 to 10. Higher approach rates mean that hit objects will be shown for a shorter period of time, thus giving less reaction time to plan ahead. Inversely, lower approach rates mean that hit objects will be shown on the screen for a longer time, allowing more time to react, but can result in an excessive amount of hit objects appearing on the screen at once. The duration of a hit object that stays visible on the screen (without mods) ranges from 1800ms at AR0 to 450ms at AR10.
```

Comment:
I think the last sentence could instead begin a new section about the timing info, containing up to line 32